vendor.conf

```diff
@@ -19,7 +19,7 @@ github.com/gorilla/mux 599cba5e7b6137d46ddf58fb1765f5d928e69604
 github.com/inconshreveable/mousetrap 76626ae9c91c4f2a10f34cad8ce83ea42c93bb75
 github.com/jmespath/go-jmespath bd40a432e4c76585ef6b72d3fd96fb9b6dc7b68d
 github.com/marstr/guid 8bd9a64bf37eb297b492a4101fb28e80ac0b290f
-github.com/satori/go.uuid f58768cc1a7a7e77a3bd49e98cdd21419399b6a3
+github.com/satori/go.uuid 75cca531ea763666bc46e531da3b4c3b95f64557
```

Comment:
Was there a reason not to use current master for this? https://github.com/satori/go.uuid/compare/75cca531ea763666bc46e531da3b4c3b95f64557...master

It's a pity we still have this dependency (as it's basically unmaintained), but looks like the Azure team doesn't want to move away from it; https://github.com/Azure/azure-sdk-for-go/issues/3158


Actually, I see master is currently on v1.2.0; https://github.com/docker/distribution/blob/bf56f348be647ad19c8204b18f0146be40db53dc/go.mod#L34

If these fixes are critical, we must make sure that master has those as well, otherwise the next release that's cut from master will have a regression

Reply:
Based on the delta you highlighted, there is nothing preventing using master if thats preferred. I just chose the highest relevant commit.

Reply:
Left a comment on the PR on master, I left a comment there, and I think  it'd be good to do the same here, so something along the lines of:

```
github.com/satori/go.uuid 75cca531ea763666bc46e531da3b4c3b95f64557 // v1.2.0-12-gb2ce238, to include 75cca531 (WS-2018-0594), which is not in a tagged release
```